uniflow-core/src/main/kotlin/io/uniflow/core/flow/ActionReducer.kt

```diff
@@ -14,8 +14,8 @@ import kotlinx.coroutines.isActive
 import kotlinx.coroutines.withContext
 
 @UseExperimental(ObsoleteCoroutinesApi::class)
-class ActionFlowScheduler(private val uiDataManager: UIDataManager, private val coroutineScope: CoroutineScope,
-    private val defaultDispatcher: CoroutineDispatcher, defaultCapacity: Int = Channel.BUFFERED) {
+class ActionReducer(private val uiDataManager: UIDataStore, private val coroutineScope: CoroutineScope,
```

Comment:
You could rename `uiDataManager` too:

```suggestion
class ActionReducer(private val uiDataStore: UIDataStore, private val coroutineScope: CoroutineScope,
```